fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -143,41 +154,22 @@ def self.run(options)
 
         UI.user_error!("Symbols on path '#{File.expand_path(dsym_filename)}' not found") if dsym_filename && !File.exist?(dsym_filename)
 
-        if options[:upload_dsym_only]
-          UI.success('Starting with dSYM upload to HockeyApp... this could take some time.')
-        else
-          UI.success('Starting with file(s) upload to HockeyApp... this could take some time.')
+        if dsym_filename
+          dsym_io = Faraday::UploadIO.new(dsym_filename, 'application/octet-stream') if dsym_filename and File.exist?(dsym_filename)
```

Comment:
This conversion should be made at a lower level, later in the call chain. I am surprised it works given that the require 'faraday' is made in the `connection` method which is called later.

Have you run that code after your last changes? Is there something that requires faraday in your project before this action?